### PR TITLE
updates to strategy

### PR DESCRIPTION
**Pull request by eugeniaft**

### CODE REVIEW

File: content/strategy-goals/strategy/code-graph/search/core.md [hunk 1]
```diff
@@ -1,38 +1,46 @@
...
 
 - [Search overall strategy](../index.md)
 - [Search Core backlog](https://github.com/sourcegraph/sourcegraph/issues?q=is%3Aopen+is%3Aissue+label%3Ateam%2Fsearch-core)
-- [Roadmap](https://github.com/orgs/sourcegraph/projects/214/views/34?filterQuery=owning-org%3A%22Code+Graph%22+type%3ARoadmap+owning-team%3A%22Search+core%22)
+- [Roadmap](https://github.com/orgs/sourcegraph/projects/214/views/21?filterQuery=quarter%3A%22FY23+Q2%22+owning-team%3A%22Search+core%22+)
 
 ## Where we are now
 
-In **FY22Q3**, we grew the Sourcegraph Cloud global index to 2.1M repositories, including all repositories with 5 stars or more. Importantly, the changes we’ve made to reach this state have not been cloud-specific, and yielded trickle-down benefits to all Sourcegraph deployments (for instance, [significant reductions](https://about.sourcegraph.com/blog/zoekt-memory-optimizations-for-sourcegraph-cloud/) in the memory usage of Zoekt, our trigram-based indexed search backend).
+Search is the foundation of the overall product and the team is focused on making search universal, fast, and relevant.
```
Comment (jasonyavorsky) [inline content/strategy-goals/strategy/code-graph/search/core.md:15]: Really nice, I like how succint and clear this is.

File: content/strategy-goals/strategy/code-graph/search/core.md [hunk 2]
```diff
@@ -1,38 +1,46 @@
...
 
-In **FY22Q4**, we conducted discovery work to better understand the bottlenecks of our search infrastructure on large monorepos (>6GB working directory). We also started growing our search index to include repositories from more non-GitHub.com and GitLab.com code hosts: for instance, you can now search [34k repositories from src.fedoraproject.org on sourcegraph.com](https://sourcegraph.com/search?q=context:global+r:%5Esrc%5C.fedoraproject%5C.org/+type:repo+count:all&patternType=literal).
+We have worked toward the goal of indexing open source code to make our search universal:
```
Comment (jasonyavorsky) [inline content/strategy-goals/strategy/code-graph/search/core.md]: Maybe worth mentioning that this has been a recent focus.

```suggestion
We have recently worked toward the goal of indexing open source code to make our search universal:
```

File: content/strategy-goals/strategy/code-graph/search/core.md [hunk 3]
```diff
@@ -1,38 +1,46 @@
...
 
-### FY23Q1
+### FY23Q2
 
 #### Goals
 
-- **Monorepo performance**: At a P75 level, gigarepo will index in less than 30 minutes, indexed searches complete in < 2s and unindexed searches complete in < 10s. Gigarepo is a representative monorepo with a HEAD working copy size of 15GB.
-- **Ranking**: We start tracking ranking quality, using selected search results as a proxy.
-- **Code host coverage**: Sourcegraph Cloud indexes public repositories globally from the most popular package hosts.
+- **Monorepo performance**: At a P75 level, synthetic gigarepo will index in less than 30 minutes, indexed searches will complete in < 2s and unindexed searches will complete in < 10s.
```
Comment (jasonyavorsky) [inline content/strategy-goals/strategy/code-graph/search/core.md:33]: Maybe worth explaining what a gigarepo is here, or using another term that more people will understand.

File: content/strategy-goals/strategy/code-graph/search/core.md [hunk 4]
```diff
@@ -1,38 +1,46 @@
...
+- **Ranking**: Understand customer pain points and establish a baseline for future improvements.
+- **Dependencies Search**: Expand support for dependencies search to Python and JVM and implement `repo:dependents(...)`.
 
 #### Details
 
-**Monorepo performance**: is a recurrent pain point for large enterprise customers. Having replicated large monorepo setups, we identified that unindexed monorepo performance is still poor and several facets of search on large monorepos cause significant load on gitserver.
+**Monorepo performance**: Is a recurrent pain point for large enterprise customers. Having replicated large monorepo setups, we identified that unindexed monorepo performance is still poor and several facets of search on large monorepos cause significant load on gitserver. To address this, we have been working on an incremental indexing prototype and a Searcher prototype to better handle unindexed searches.
```
Comment (jasonyavorsky) [inline content/strategy-goals/strategy/code-graph/search/core.md]:
```suggestion
**Monorepo performance**: This is a recurrent pain point for large enterprise customers. Having replicated large monorepo setups, we identified that unindexed monorepo performance is still poor and several facets of search on large monorepos cause significant load on gitserver. To address this, we have been working on an incremental indexing prototype and a Searcher prototype to better handle unindexed searches.
```

**Review by jasonyavorsky — APPROVED** [review]
Looks great! Please leave it open for a few days for more cross-team inputs/questions.

File: content/strategy-goals/strategy/code-graph/search/core.md [hunk 5]
```diff
@@ -1,38 +1,46 @@
...
+**Ranking**: As a first step towards improving the ranking of our search results, we will start tracking the quality of search results using the index of user-selected results as a proxy metric. Having this tracking in place will help measure the success of future improvements. We will also conduct discovery of customers needs and pain points to drive the areas of ranking we choose to focus on.
 
-**Code host coverage**: as we seek to keep expanding the code we index to include more non-GitHub.com or GitLab.com code hosts, we will add support for package host integrations (PyPi, Rubygems, NuGet, Crates, proxy.golang.org). This will not only increase our code host coverage, but also be a stepping stone towards unblocking [use cases based on the dependency graph of repositories](https://docs.google.com/document/d/1SkM8CG0IksvPEKRBRVLKipiRJTopx6Vq_hSWRJ9NyKs/edit#).
+**Dependencies Search**: We will expand support for dependencies search to include Python and JVM and implement `repo:dependents(...)` allowing users to search which repositories depend on a specific package. This will be particularly relevant for the fixing security vulnerabilities use case.
```
Comment (jjeffwarner) [inline content/strategy-goals/strategy/code-graph/search/core.md:43]: Would it be worth adding a sentance at the end to tie the transition from code host coverage into dependency search? Something like 'This was formerly code host coverage'?

File: content/strategy-goals/strategy/code-graph/search/core.md (hunk 3 above)
Comment (yinquanteo) [reply content/strategy-goals/strategy/code-graph/search/core.md:33]: A direct link could work too

File: content/strategy-goals/strategy/code-graph/search/core.md [hunk 6]
```diff
@@ -1,38 +1,46 @@
...
+**Ranking**: As a first step towards improving the ranking of our search results, we will start tracking the quality of search results using the index of user-selected results as a proxy metric. Having this tracking in place will help measure the success of future improvements. We will also conduct discovery of customers needs and pain points to drive the areas of ranking we choose to focus on.
 
-**Code host coverage**: as we seek to keep expanding the code we index to include more non-GitHub.com or GitLab.com code hosts, we will add support for package host integrations (PyPi, Rubygems, NuGet, Crates, proxy.golang.org). This will not only increase our code host coverage, but also be a stepping stone towards unblocking [use cases based on the dependency graph of repositories](https://docs.google.com/document/d/1SkM8CG0IksvPEKRBRVLKipiRJTopx6Vq_hSWRJ9NyKs/edit#).
```
Comment (yinquanteo) [inline content/strategy-goals/strategy/code-graph/search/core.md:35]: Are we completely dropping package host integrations. I know we have fedora set up, are we not planning to pursue the rest?

**Review by yinquanteo — APPROVED** [review]

File: content/strategy-goals/strategy/code-graph/search/core.md (hunk 6 above)
Comment (jjeffwarner) [reply content/strategy-goals/strategy/code-graph/search/core.md:35]: We are getting a lot of that via dependency search.

**Pull request merged**